### DISCUSSION TITLE
"•NET" looks weird. Why not just ".NET"?

### DISCUSSION THREAD

**Discussion by eyalch** [starship/starship#2423] [General]
The official writing of .NET is, well, ".NET", so why not write it like that?

**Comment by andytom** [comment]
Hi @eyalch, I think this was just a stylistic choice by the original author of the module. I am not a dot net developer so not sure what is expected but if it is more normal to write `.NET` please feel free to open a PR to change this.

**Comment by eyalch** [comment]
Will do. Thanks!
